### ISSUE TITLE
Google Drive, Implimentation

### ISSUE THREAD

**Issue by alexaaaaaander** [karaposu/comfyui-on-cloud#7]
I've got a tonnnn of checkpoints, controlnets, loras, etc already existing in my Gdrive.. is there a way a novice (like myself) can link them??

**Comment by karaposu** [comment]
Yeah, easiest way is to make them public from sharing settings. This way they can be download via curl command.

**Comment by alexaaaaaander** [comment]
Any chance I can find a step-by-step process? I’m very much a novice 

> Yeah, easiest way is to make them public from sharing settings. This way they can be download via curl command.

**Comment by karaposu** [comment]
chatgpt is your friend. Just make sure you descripe your issue clearly.  Write like below and ask where you dont understand: 

"I have some model files in my google drive. I also have a linux VM. What is the easiest way to download these files to specific folder in my VM? (i dont want to use sth complex) "
